django_app/report_a_suspected_breach/templates/report_a_suspected_breach/form_steps/check_your_answers.html

```diff
@@ -331,19 +331,19 @@ <h3 class="govuk-heading-s" style="float:left">End-users</h3>
         {% endif %}
     </dl>
     {% if form_data.were_there_other_addresses_in_the_supply_chain.were_there_other_addresses_in_the_supply_chain  == "yes" %}
-        <h2 class="govuk-heading-m">Other addresses in the supply chain</h2>
+        <h2 class="govuk-heading-m">Other people or businesses involved in this trade</h2>
```

Comment:
change this to an h3

```suggestion
        <h3 class="govuk-heading-m">Other people or businesses involved in this trade</h3>
```